### DISCUSSION TITLE
How to share/download multi remote images to Photos/Gallery

### DISCUSSION THREAD

**Discussion by anhtukhtn** [ionic-team/capacitor#4005] [Q&A]
Hi, I spent a few days for searching in Capacitor docs and others, but can't find any way to save/download/share list of remote images to Photos/Gallery in mobile.
The Share Plugin(https://capacitorjs.com/docs/apis/share) just can share one url.

React Native can share multi urls (https://github.com/react-native-share/react-native-share)

How to archive something like  share/download multi urls with Capacitor?

**Comment by imhoffd** [comment]
I don't think it's possible today. I created an issue for a feature request: https://github.com/ionic-team/capacitor-plugins/issues/173

**Reply by anhtukhtn** [reply]
Thank you very much for your response

**Reply by anhtukhtn** [reply]
More information, I found https://ionic.io/docs/premier-plugins/social-sharing, they have saveToPhotoAlbum and share files
